_layouts/index.html

```diff
@@ -8,7 +8,18 @@
     {% assign latestRelease = site.data.releases[site.current_release_index] %}
     <h1>Wild<strong>Fly</strong></h1>
     <h2>A powerful, modular, & lightweight application server that helps you build amazing applications.</h2>
-    <h3>Now available: <span style="font-weight: 100;">Wild</span><strong>Fly</strong> {{latestRelease.version_shortname}}</h3>
+
+    {% for link in latestRelease.link %}
+      {% if link.name == "Release Notes" %}
+        {% for item in link.items %}
+          {% if item.format == "Notes" %}
+            {% assign releaseNoteLink = item.url %}
+          {% endif %}
+        {% endfor %}
+      {% endif %}
+    {% endfor %}
+
+    <h3>Now available: <a href="{{ releaseNoteLink }}"><span style="font-weight: 100;">Wild</span><strong>Fly</strong> {{latestRelease.version_shortname}}</a></h3>
```

Comment:
With this I think we need a rule that the "Release Notes" data for future versions in release.yaml can never be a link to the oddly formatted JIRA releases notes. It must be to a blog post or to the GitHub release page.

I think such a rule is ok, but needs socialization as our requirements around this have been changing a bit in the last year and people coordinating releases may not be aware of things.

I added release process content related to this: https://docs.google.com/document/d/1Bjefo56-J_8PtPxTbHCiE-M3TeelHjxKppaJMmu6HSE/edit#bookmark=id.byujw232n9l